### PR TITLE
fixes for nodejs_l7 python example due to google_containers project deprecated to cos-cloud

### PR DESCRIPTION
**Pull request by daverick**
- change project from google_containers to cos-cloud
- change the key to 'gce-container-declaration' for the container manifest
- add service account default with scope logging.write to the instances
- fix the Pod and container name by casting the unicode value to string

### CODE REVIEW

File: examples/v2/common/python/container_vm.py [hunk 1]
```diff
@@ -69,7 +69,11 @@ def GenerateConfig(context):
           'network': GlobalComputeUrl(context.env['project'],
                                       'networks',
                                       'default')
-          }]
+          }],
+        'serviceAccounts': [{
```
Comment (vagababov) [inline examples/v2/common/python/container_vm.py]: why is this change required?

Comment (daverick) [reply examples/v2/common/python/container_vm.py]: the container are not starting without it. They produce an error like this:
`docker: Error response from daemon: Failed to initialize logging driver: unable to connect or authenticate with Google Cloud Logging: Post https://logging.googleapis.com/v1beta3/projects/qwiklabs-gcp-f29e3dcb8073a1d0/logs/gcplogs-docker-driver/entries:write?alt=json: metadata: GCE metadata "instance/service-accounts/default/token" not defined.`
